demo/components/tree-view-demo.tsx

```diff
@@ -37,15 +40,62 @@ function SelectedItem({selectedItem}: any) {
 export interface TreeViewDemoState {
     selectedItem: TreeItemData | undefined;
     focusedItem: TreeItemData | undefined;
+    selectedItemCustom: TreeItemData | undefined;
+    focusedItemCustom: TreeItemData | undefined;
 }
 
+export const CustomItem: React.SFC<TreeItemProps> =
```

Comment:
This demo is non relatable. Lets think of a more real world example?

Reply:
a food menu is relatable in my taste, especially the desserts part :)